packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -263,16 +263,22 @@ contract OptimisticGovernor is Module, Lockable {
      * @param _proposalId the id of the proposal being deleted.
      */
     function deleteProposal(uint256 _proposalId) public onlyOwner {
+        // Check that proposal exists and was not already deleted.
+        require(proposalHashes[_proposalId] != "", "Proposal does not exist");
+
         delete proposalHashes[_proposalId];
-        emit ProposalDeleted(_proposalId);
+        emit ProposalDeleted(_proposalId, msg.sender, "DeletedByOwner");
     }
 
     /**
      * @notice Method to allow anyone to delete a proposal that was rejected.
      * @param _proposalId the id of the proposal being deleted.
-     * @param _originalTime the id of the proposal being deleted.
+     * @param _originalTime the time the proposal was made.
      */
     function deleteRejectedProposal(uint256 _proposalId, uint256 _originalTime) public {
+        // Check that proposal exists and was not already deleted.
+        require(proposalHashes[_proposalId] != "", "Proposal does not exist");
```

Comment:
This (and the other one) were fixes for a different issue, as were a few other changes in this PR, but I wanted to stay consistent with the audit fixes that were happening in parallel. Basically, this should work cleanly with the other fixes if we decide to merge it in, but the other fixes don't depend on the changes in this PR and could be merged separately, since we aren't sure if this is the right direction yet.